parent/pom.xml

```diff
@@ -1066,53 +1070,136 @@
 			</properties>
 		</profile>
 
+		<!-- JBIDE-23206
+			 To force a local build of a pull request to use current date/time when generating plugins and features,
+			 instead of using jgit timestamps (which could be in the past, and therefore result in stuff that Eclipse
+			 thinks is OLDER that a current N build, use this "R" profile:
+			 mvn verify -PR
+		-->
 		<profile>
-			<id>test-on-installed-app</id>
```

Comment:
test-on-installed-app is basically the same profile as preinstalled-devstudio but preinstalled-devstudio has a bit updated configuration. 

Also I did not add R profile - thats just github incorrectly showing changes (take a look at file here https://github.com/rawagner/jbosstools-build/blob/a7deb18c724626bc0a0d2f5a2b181337f86ad4c4/parent/pom.xml - just one R profile )